crates/ruff_linter/src/rules/flake8_commas/rules/trailing_commas.rs

```diff
@@ -365,7 +367,7 @@ pub(crate) fn trailing_commas(
                 // removing any brackets in the same linter pass - doing both at the same time could
                 // lead to a syntax error.
                 let contents = locator.slice(missing_comma.1);
-                diagnostic.set_fix(Fix::automatic(Edit::range_replacement(
+                diagnostic.set_fix(Fix::always_applies(Edit::range_replacement(
```

Comment:
I'm leaning towards explicit since there was some confusion expressed about the previous methods. I feel like with a bare "always" it's unclear what it refers to and could be confused with the the availability?